envVariables.js

```diff
@@ -33,6 +33,10 @@ let enviromentVariables = {
 		message: 'Enable/Disable kafka communications',
 		optional: false,
 	},
+	ELEVATE_ENTITY_SERVICE_BASE_URL: {
+		message: 'Please specify the value for e.g. 5000',
```

Comment:
Elevate-entity service base URL is required @Prajwal17Tunerlabs change message

Reply:
message changed